requirements-dev.txt

```diff
@@ -10,3 +10,4 @@ setuptools_scm
 sphinx
 sphinx_rtd_theme
 towncrier
+hypertools
```

Comment:
Example requirements are not development requirements, therefore this change is not necessary.